testing/cypress/integration/restaurant_routing.spec.js

```diff
@@ -0,0 +1,45 @@
+
```

Comment:
Hey, great work with the new tests. I was just wondering if it would be more helpful if we had a block comment at the top of each test that described what it did and why we are testing it. I know there are already describe statements but explaining tests will be helpful for other developers/viewer who want to know what the test does without reading source code. Overall awesome stuff.

Reply:
Hi Nidhineshnand, I was planning to have a separate doc for mapping the test file with the test case. Do you think this will helpful for others to understand?

Reply:
That would definitely be helpful but it might be hard for others to find the file and switching between the file and tests might become tedious. I would be happy to have like some sort of Javadoc type of file where the docs are directly extracted from comments.

Reply:
Had a quick search, seems JS has similar doc control like Javadoc, but from cypress website it doesn't have similar example. So I added a block comment which only contain scenario description, hopefully this is enough.

Reply:
Seems good for now